src/io/rkn/conformity.clj

```diff
@@ -1,8 +1,23 @@
 (ns io.rkn.conformity
-  (:use [datomic.api :only [q db] :as d]
-        [clojure.java.io :as io]))
+  (:require [datomic.api :refer [q db] :as d]
+            [clojure.java.io :as io]))
 
 (def default-conformity-attribute :confirmity/conformed-norms)
+(def conformity-ensure-norm-tx :conformity/ensure-norm-tx)
+
+(def ensure-norm-tx-txfn
+  "transaction function to ensure each norm tx is executed exactly once"
```

Comment:
Capitalization. (Though I'll modify it post-merge, if you don't get too it first.)